dss/subscriptions_v2/__init__.py

```diff
@@ -22,64 +18,38 @@ class SubscriptionData:
     ATTACHMENTS = 'attachments'
 
 
+subscription_db_table = f"dss-subscriptions-v2-{{}}-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+
 def put_subscription(doc: dict):
-    dynamodb.put_item(
-        TableName=_ddb_table_template.format(doc[SubscriptionData.REPLICA]),
-        Item={
-            'hash_key': {
-                'S': doc[SubscriptionData.OWNER]
-            },
-            'sort_key': {
-                'S': doc[SubscriptionData.UUID]
-            },
-            'body': {
-                'S': json.dumps(doc)
-            },
-        }
-    )
+    dynamodb.put_item(table=subscription_db_table.format(doc[SubscriptionData.REPLICA].name),
+                      key1=doc[SubscriptionData.OWNER],
+                      key2=doc[SubscriptionData.UUID],
+                      value=json.dumps(doc))
 
-def get_subscription(replica: Replica, owner: str, uuid: str) -> dict:
-    db_resp = dynamodb.get_item(
-        TableName=_ddb_table_template.format(replica.name),
-        Key={
-            'hash_key': {
-                'S': owner
-            },
-            'sort_key': {
-                'S': uuid
-            }
-        }
-    )
-    item = db_resp.get('Item')
+
+def get_subscription(replica: Replica, owner: str, uuid: str):
+    item = dynamodb.get_item(table=subscription_db_table.format(replica.name),
+                             key1=owner,
+                             key2=uuid)
     if item is not None:
         return json.loads(item['body']['S'])
     else:
         return None
 
+
 def get_subscriptions_for_owner(replica: Replica, owner: str) -> list:
-    db_resp = dynamodb.query(
-        TableName=_ddb_table_template.format(replica.name),
-        KeyConditionExpression="#hash_key=:owner",
-        ExpressionAttributeNames={'#hash_key': "hash_key"},
-        ExpressionAttributeValues={':owner': {'S': owner}}
-    )
-    subscriptions = [json.loads(item['body']['S']) for item in db_resp['Items']]
-    return subscriptions
+    items = dynamodb.get_primary_key_items(table=subscription_db_table.format(replica.name),
+                                           key=owner)
+    return [json.loads(item['body']['S']) for item in items]
```

Comment:
Can we move `['body']['S']` into `dynamodb.get_primary_key_items`?

Reply:
Alright.